dotnetUno5FrameBuffer/Dockerfile

```diff
@@ -7,7 +7,7 @@ ARG IMAGE_ARCH=
 ##
 # Base container version
 ##
-ARG BASE_VERSION=rc-8.0
```

Comment:
I'm reverting this, dotnetUno5FrameBuffer is a community one and we will start from the fresh